{
	"schema_version": "1.6.0",
	"id": "RHSA-2021:0717",
	"modified": "2024-09-13T18:17:16Z",
	"published": "2024-09-13T18:17:16Z",
	"related": [
		"CVE-2020-14781",
		"CVE-2020-14782",
		"CVE-2020-14803",
		"CVE-2020-27221",
		"CVE-2020-2773"
	],
	"summary": "Red Hat Security Advisory: java-1.8.0-ibm security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm-demo",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm-demo"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm-devel",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm-devel"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm-jdbc",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm-jdbc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm-plugin",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm-plugin"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:rhel_extras:7",
				"name": "java-1.8.0-ibm-src",
				"purl": "pkg:rpm/redhat/java-1.8.0-ibm-src"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:1.8.0.6.25-1jpp.1.el7"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:0717.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2021:0717"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#critical"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1823224"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1889274"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1889290"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1889895"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1928555"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2021/rhsa-2021_0717.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-2773"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-2773"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-2773"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-14781"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-14781"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14781"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-14782"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-14782"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14782"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-14803"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-14803"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-14803"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2020-27221"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2020-27221"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-27221"
		}
	]
}
